relationships/synthesis/APM-APPLICATION-to-INFRA-MSSQLINSTANCE.yml

```diff
@@ -1,12 +1,15 @@
 relationships:
   - name: apmApplicationCallsInfraMSSQLInstance
-    version: "1"
+    version: "2"
```

Comment:
```suggestion
    version: "1"
```
This attribute references the [version of the rule definition format](https://github.com/newrelic/entity-definitions/blob/main/docs/relationships/relationship_synthesis.md#version), which is still v1.